server/public/model/manifest.go

```diff
@@ -116,6 +116,9 @@ type PluginSettingsSection struct {
 
 	// If true, the section will load the custom component registered using `registry.registerAdminConsoleCustomSection`
 	Custom bool `json:"custom" yaml:"custom"`
+
+	// If true and Custom = true, the settings defined under this section will still render as fallback (unless of custom type) when the plugin is disabled.
```

Comment:
this is a little confusing, imo. It's almost like you're saying `if custom && fallback, render unless custom` (which conflicts with with the first condition of the if stmt). Maybe:
```suggestion
	// If true and Custom = true, the settings defined under this section will still render as fallback (unless the individual setting is type 'custom') when the plugin is disabled.
```